### ISSUE TITLE
jpeg images are not exported if RAW+JPEG pair and jpeg is selected as master on Mojave

### ISSUE THREAD

**Issue by RhetTbull** [RhetTbull/osxphotos#229]
On Mojave: `osxphotos export ~/Desktop/export -V` does not export the jpeg image of a RAW+JPEG pair if the RAW image is selected as original in Photos:

```
Using last opened Photos library: /Users/rhet/Pictures/Test-10.14.6.photoslibrary
Using export database /Users/rhet/Desktop/export/.osxphotos_export.db
Exporting 12 photos to /Users/rhet/Desktop/export...
...
Exporting IMG_1997.cr2 (IMG_1997.cr2) as IMG_1997.cr2
Exported /Users/rhet/Desktop/export/IMG_1997.cr2
Exported /Users/rhet/Desktop/export/IMG_1997.cr2
...
Exporting IMG_1994.JPG (IMG_1994.JPG) as IMG_1994.JPG
Exported /Users/rhet/Desktop/export/IMG_1994.JPG
Exported /Users/rhet/Desktop/export/IMG_1994.cr2
...
Exported: 16 photos
Elapsed time: 0.417 seconds
```

Also looks like the raw image is exported twice.

**Comment by RhetTbull** [comment]
```
>>> photo = photosdb.get_photo("AcxIpfolT3KU2Ge84VG3yQ")
>>> photo.has_raw
True
>>> photo.path
'/Users/rhet/Pictures/Test-10.14.6.photoslibrary/Masters/2020/10/05/20201005-041506/IMG_1997.cr2'
>>> photo.path_raw
'/Users/rhet/Pictures/Test-10.14.6.photoslibrary/Masters/2020/10/05/20201005-041506/IMG_1997.cr2'
>>>
```

**Comment by RhetTbull** [comment]
I'm not sure what the best way to represent these images is.  Photos treats them differently -- In Photos 4 for example, RKVersion is updated when the user selects "Use RAW as original" and Photos displays the "R" badge on the photo.  One approach might be for `osxphotos` to always treat them the same but use `PhotoInfo.raw_original` flag to indicate which is the original.  E.g. in this case:
`.path` would always be the jpeg
`.path_raw` would always be the raw
`.has_raw` == True
`.raw_original` = True or False

**Comment by RhetTbull** [comment]
Fixed in v0.35.0
